og_access/src/OgAccessBundleFormAlter.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Drupal\og_access;
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Form\FormStateInterface;
```

Comment:
```suggestion
use Drupal\Core\Form\FormStateInterface;
use Drupal\Core\StringTranslation\StringTranslationTrait;
```

( ! ) Fatal error: Trait 'Drupal\og_access\StringTranslationTrait' not found in /web/modules/contrib/og/og_access/src/OgAccessBundleFormAlter.php on line 14
--